DNA_RNA_classes_first_commit.py

```diff
@@ -0,0 +1,44 @@
+class Dna(str):
+    
+    
+    def __init__(self, dna_seq):  
+        self.dna_seq = dna_seq

```

Comment:
If you applied "upper" string method here in init part, you would escape repeating it in methods. Also, your list of codes will contain only uppercase letters.

Reply:
Do I need to add this as "self = self.upper()"? The problem is that I tried, and the code did not work properly, i.e. if did not convert lowercase letters to uppercase letters.

Reply:
I mean, self.dna_seq = dna_seq.upper()

Reply:
This does not work for me, I do not know why

Reply:
Works now, thanks!